vllm/model_executor/layers/sampler.py

```diff
@@ -186,6 +186,12 @@ def __init__(self):
         # speculative decoding.
         self.include_gpu_probs_tensor = False
         self.should_modify_greedy_probs_inplace = False
+        from vllm.config import get_current_vllm_config
```

Comment:
I think that we should pipe this parameter down though the constructor rather than querying the global config